ostd/src/bus/pci/capability/mod.rs

```diff
@@ -6,13 +6,16 @@
 
 use alloc::vec::Vec;
 
-use self::{msix::CapabilityMsixData, vendor::CapabilityVndrData};
+#[cfg(target_arch = "x86_64")]
+use self::msix::CapabilityMsixData;
+use self::vendor::CapabilityVndrData;
 use super::{
     cfg_space::{PciDeviceCommonCfgOffset, Status},
     common_device::PciCommonDevice,
     PciDeviceLocation,
 };
 
+#[cfg(target_arch = "x86_64")]
```

Comment:
Instead of using `#[cfg(target_arch = "x86_64")]`, it may be better to use `#[cfg(not(target_arch = "loongarch64"))]`.
This is because other architectures, such as RISC-V, may still want to support MSI-X.

Reply:
Oh, this PR was deprecated, and maybe #7 is a better way to resolve the msix issue.

Reply:
Are you planning to close this PR? It appears that #7 is still in draft. Is #7 ready yet?

Reply:
No, I was still trying to find a way to enable the feature for x86_64. But it should be ready soon

Reply:
#7 was ready, should I close this PR now?

Reply:
Sure, you can close this PR.